showpass/Showpass-E2E-tests
# TA-43 Verify filters on the 'all events' page

**Description:**
This spec will be verifying that events can be filtered based on the filter criteria provided.

Steps:
- Navigate to [Search Experiences](https://beta.showpass.com/s/events/all/) 
- Apply and verify the following search criteria

Case 1:
Enter a valid event name and verify the filter
Case 2:
Invalid event name and the corresponding message stating “No events available with your search criteria.”
Case 3:
Valid event name and date range
Case 4:
Filter by Category
Case 5:
Filter by a valid tag
Case 6:
Filter by an invalid tag
Case 7:
Verify the ‘Clear all filters’ button functionality

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-43

**How to test:**
Run the following test suite and ensure the tests pass.
cypress/e2e/events/verify-filters.cy.js

**Test execution results:**

<img width="953" alt="TA-43-all-tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/3e6d983d-58a8-4947-bb5c-9d0f5e70d517">
